Tasks/AzureCLI/azureclitask.ts

```diff
@@ -130,8 +130,9 @@ export class azureclitask {
         if (endpointAuth.scheme === "Certificate") {
             var bytes = endpointAuth.parameters["certificate"];
             var subscriptionId:string = tl.getEndpointDataParameter(connectedService, "SubscriptionId", true);
+            var serviceManagementUrl:string = tl.getEndpointUrl(connectedService, false);
             const publishSettingFileName:string = path.join(os.tmpdir() ,"subscriptions" + new Date().getTime() + ".publishsettings");
```

Comment:
a default service URL used to be present in older endpoints. so we should be fine.